packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/README.md

```diff
@@ -48,7 +49,9 @@ subject.locations = [
 
 ### JSON file
 
-The JSON file is a very, very basic data object consisting of an array of numbers for each axis.
+The JSON file can take two different shapes depending on if the data is a single series or multiple series. 
+
+The single series JSON shape is a very, very basic data object consisting of an array of numbers for each axis:
```

Comment:
Is there a reason that the single series data isn't an array containing just one series?

Reply:
For backwards compatibility for JSON shape that the data takes for the LCV for TESS. The long goal would be to replace the LCV there with this since this is being written with tests and doesn't have the potential performance and data issue of having two different libraries update the DOM. That remains to be seen since work would need to be done to add brushes for doing the annotations. The VSLCV will not be making annotations like this, so that's a ways off.

Reply:
👍